lib/simp/beaker_helpers/network.rb

```diff
@@ -0,0 +1,62 @@
+module Simp; end
+module Simp::BeakerHelpers; end
+
+module Simp::BeakerHelpers::Network
+
+  # @returns array of IPV4 networks configured on a host or nil
+  #   if none can be determined
+  #
+  # NOTE:
+  # - Result can be used for simp_options::trusted_nets in a test.
+  # - Uses the modern 'networking' fact from the puppet-agent RPM.
+  #
+  # @param host [Host] the host whose networks info is to be retrieved
+  #
+  def host_networks(host)
```

Comment:
Should we be able to pass these a set of interfaces so that we can specify which one is internal vs. external?

Unfortunately, this is really only effective in the VirtualBox case since the cloud providers only end up with a single public interface that they use for everything.